lib/puppet/functions/simplib/bracketize.rb

```diff
@@ -0,0 +1,63 @@
+# Add brackets to IP addresses and `Arrays` of IP
+# addresses based on the rules for bracketing
+# IPv6 addresses.
+#
+# Ignore anything that does not look like an IPv6 address.
+#
+Puppet::Functions.create_function(:'simplib::bracketize') do
+
+  # @param ipv6 The ipv6 to bracketize
```

Comment:
This param much match the parameter in the dispatch.  In this case 'ipv6' -> ipaddr

Reply:
thanks for the catch, was a holdover from when I was deciding on a name